pkg/controller/master/backup/backupTaget.go

```diff
@@ -36,6 +39,18 @@ const (
 	longhornBackupTargetSecretSettingName = "backup-target-credential-secret"
 )
 
+const (
+	// StaticCredentialsName provides a name of Static provider
+	StaticCredentialsName = "StaticCredentials"
```

Comment:
The const is unused; 
Is this better: https://github.com/aws/aws-sdk-go-v2/blob/b7d8e15425d2f86a0596e8d7db2e33bf382a21dd/config/example_test.go#L44

Reply:
updated